rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -72,6 +73,10 @@ def ensure_global(name, default):
 
 ensure_global('rosdoc2_settings', {{}})
 ensure_global('extensions', [])
+ensure_global('project', "{package_name}")
+ensure_global('copyright', "{year}, {package_licenses}")
```

Comment:
The copyright is usually the year and the copyright holder(s), not the year and the types of licenses. Unfortunately this isn't something that is captured in the package.xml there can be one or many copyright holders. I'd recommend skipping this one instead of doing a partial job. Also it's not clear to me if the current year is not technically right for this if there's no edits in the current year.

Reply:
makes sense to me, i removed it for now